### ISSUE TITLE
quaternion.toRotationMatrix() does not work

### ISSUE THREAD

**Issue by gabrielebndn** [stack-of-tasks/eigenpy#94]
These lines
```
import eigenpy
q = eigenpy.Quaternion(1.,0.,0.,0.)
q.toRotationMatrix()
```
result in the following error
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-676a92d9f600> in <module>()
----> 1 q.toRotationMatrix()

AttributeError: 'Quaternion' object has no attribute 'toRotationMatrix'
```

I do not understand this, as I can clearly see in https://github.com/gabrielebndn/eigenpy/blob/72f57d877e2b9fb7b0d2d216ce947ad62073286f/include/eigenpy/quaternion.hpp#L106 that `toRotationMatrix` is exposed, correctly, it seems.
What is weirder is that `ipython` actually *suggests* `toRotationMatrix` as an available method and auto-completes it, but then it gives an error as if it did not know it if you actually try to use it. What is going on?

This is not super-important as the same functionality is provided by the method `matrix()`. The duplication comes from Eigen itself. The methods are inherited from  [RotationBase](https://eigen.tuxfamily.org/dox/classEigen_1_1RotationBase.html#title14), where the documentation seems to suggest that the preferred method should be `toRotationMatrix`, while `matrix` was only "added to be conform with the Transform class' naming scheme." Further, in [QuaternionBase](https://eigen.tuxfamily.org/dox/classEigen_1_1QuaternionBase.html) only `toRotationMatrix` is documented

**Comment by jcarpent** [comment]
I've found the bug. I will open a PR fixing the issue.
@gabrielebndn Thanks for the useful feedback.

**Comment by jcarpent** [comment]
Fixed by #95.
